amethyst_audio/src/output.rs

```diff
@@ -17,7 +19,6 @@ use crate::{sink::AudioSink, source::Source, DecoderError};
 /// A speaker(s) through which audio can be played.
 ///
 /// By convention, the default output is stored as a resource in the `World`.
-#[derive(Clone, Eq, PartialEq)]
```

Comment:
`Device` is `!Clone + !PartialEq`, but it's not used anywhere anyway.

Reply:
Eq/PartialEq doesn't really seem meaningful here anyways.